### ISSUE TITLE
Process.send is not function 

### ISSUE THREAD

**Issue by emildd** [Zimtir/SENT-template#114]
I have problem when open blog content and refresh browser at `http://localhost:3000/blog/how-to-use-sapper`

step: 
* `npm run build `
* `node __sapper__/build`
* open url with hard refresh (`cmd` + `shift` + `R`)  at `http://localhost:3000/blog/how-to-use-sapper`

* console results: 
```
> Finished in 13.3s. Type node __sapper__/build to run the app.
✨  Done in 13.62s.
Starting server on port 3000
/status
{ path: '/status', status: true }
/endpoints
{ path: '/endpoints', status: true }
Service started on port 3000. 2020-12-09T04:01:32.866Z
TypeError: process.send is not a function
    at ServerResponse.res.end (/Users/emylmaulana/KMB/SENT-template/__sapper__/build/server/server.js:3093:33)
    at get$1 (/Users/..../SENT-template/__sapper__/build/server/server.js:452:9)
```

**Comment by Zimtir** [comment]
@emildd, thank you for the report!

As I see, it's issues on parts of Sapper

### Solution:

  - remove SAPPER_EXPORT from environment variables

It will be removed in repository at this PR: https://github.com/Zimtir/SENT-template/pull/88

### Details about issue:

Code at Sapper that throws issue:

![image](https://user-images.githubusercontent.com/32175240/104103578-f3552800-52b3-11eb-9d47-36f5b475f1ba.png)

Basically, they forgot add checker for process.send like it was provided here

![image](https://user-images.githubusercontent.com/32175240/104103610-1c75b880-52b4-11eb-9273-fcae68f0cb76.png)
